src/logmessage.cpp

```diff
@@ -108,7 +115,19 @@ namespace g3 {
       return out;
    }
 
-
+   std::string LogMessage::message() const {
+      std::string msg;
+      if (!_wmessage.empty())
+      {
+#if !defined(__GNUC__) || (__GNUC__ >= 5)
+         std::wstring_convert<std::codecvt_utf8<wchar_t>, wchar_t> convert;
+         msg.append(_message.append(convert.to_bytes(_wmessage)));
```

Comment:
So this means that although we have LOGW etc the ```message```` function is only implemented for the ```std::wstring``` for non-GNU. If for some reason someone want to use LOGW on a GNUC it essentially will render the ```LOGW``` call meaningless, right?